balrogscript/src/balrogscript/script.py

```diff
@@ -98,9 +98,9 @@ def schedule(task, config, auth0_secrets):
         task["payload"]["version"],
         task["payload"]["build_number"],
         task["payload"]["publish_rules"],
-        task["payload"]["force_fallback_mapping_update"] or False,
         task["payload"]["release_eta"] or None,  # Send None if release_eta is ''
-        task["payload"]["background_rate"] or None,
+        task["payload"].get("force_fallback_mapping_update", False),
+        task["payload"].get("background_rate"),
```

Comment:
Looks like you're changing the order these args are passed - that's probably going to break something :)

Reply:
You're right, mea culpa. Fixed now.